mappings/net/minecraft/entity/EntityLike.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_5568 net/minecraft/entity/EntityLike
+	COMMENT A superinterface of {@link Entity}.
+	METHOD method_24515 getBlockPos ()Lnet/minecraft/class_2338;
+		COMMENT The entity's location in the world.
+	METHOD method_31744 setChangeHandler (Lnet/minecraft/class_5569;)V
```

Comment:
`ChangeHandler` feels quite generic. Maybe something involving `Tracker`? Still quite generic, but at least implies that it's involving tracking entities around as they move / are removed.